examples/node/vector_search_example.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+import { Decoder, GlideClient, GlideClusterClient, GlideFt, FtSearchOptions,
+    FtSearchReturnType, Logger, VectorField} from "@valkey/valkey-glide";
```

Comment:
Please consider run linter command `npx prettier -w .` under `example/node` to keep consistency with other code in the repo, like the format of this import section, as linter is not enforced on example code.